final/bitcoind/entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+set -e
+
+cp -u /bitcoin.conf $BITCOIN_DATA/
+
+if [ $(echo "$1" | cut -c1) = "-" ]; then
+  echo "$0: assuming arguments for bitcoind"
+
+  set -- bitcoind "$@"
```

Comment:
I think it's clearer to specify the command on line 22 - `exec bitcoind "$@"`

Reply:
so replacing this `set` with `exec` directly?

Reply:
I think we are talking about the same thing, but I can't tell for sure without the actual change.

Reply:
`exec bitcoin "$@"`

Reply:
wait but we have to also provide the `datadir` and `conf` file if we change to just `bitcoind` that won't do it.

Reply:
sure